tests/cypress/tests-e2e/featurePreSelection.cy.js

```diff
@@ -26,24 +55,36 @@ describe('Testing the feature selection in the URL', () => {
         cy.intercept('**/MapServer/**/htmlPopup**', {
             fixture: 'html-popup.fixture.html',
         })
+        // add intercept for all features, and allow their Ids to be used in tests
+        cy.fixture('features.fixture.json').then((jsonResult) => {
+            const features = [...jsonResult.results]
+            const featuresIds = features.map((feature) => feature.id.toString())
+            cy.wrap(features).as('features')
+            cy.wrap(featuresIds).as('featuresIds')
+            features.forEach((feature) => {
+                cy.intercept(`**/MapServer/${feature.layerBodId}/${feature.id}`, {
+                    results: feature,
+                })
+            })
+        })
     })
 
     context('Startup', () => {
-        describe('Checks that when given a parameter, we select the features', () => {
-            it('Select a few features and shows the tooltip in its correct spot', () => {
-                cy.fixture('features.fixture.json').then((jsonResult) => {
-                    const features = [...jsonResult.results]
-                    features.forEach((feature) => {
-                        cy.intercept(`**/MapServer/${feature.layerBodId}/${feature.id}`, {
-                            results: feature,
-                        })
-                    })
-                    const featuresIds = features.map((feature) => feature.id.toString())
-                    cy.goToMapView({
-                        layers: `${features[0].layerBodId}@features=${featuresIds.join(':')}`,
-                    })
-                    checkFeatures(featuresIds)
-                })
+        describe('Checks that when given a parameter, we select the features', function () {
+            it('Select a few features and shows the tooltip does not appear when featureInfoPosition is not specified', () => {
+                goToMapViewWithFeatureSelection()
+                checkFeatures()
+                checkFeatureInfoPosition(FeatureInfoPositions.NONE)
+            })
+            it('Shows the tooltip in its correct position when set to default (bottom Panel on Phone)', function () {
+                goToMapViewWithFeatureSelection(FeatureInfoPositions.DEFAULT)
+                checkFeatures()
+                checkFeatureInfoPosition(FeatureInfoPositions.DEFAULT)
+            })
+            it('Shows the tooltip on the map when featureInfo is set to tooltip, and handle strange cases', function () {
+                goToMapViewWithFeatureSelection('TOoLtIp')
+                checkFeatures()
+                checkFeatureInfoPosition(FeatureInfoPositions.TOOLTIP)
```

Comment:
Do all the tests in one `it()`. `it()` are costly in term of cost in cypress cloud and performance